{{cookiecutter.project_slug}}/ansible/production_inventory

```diff
@@ -6,6 +6,7 @@ build_user = buildout
 project_slug = {{ cookiecutter.project_slug }}
 package_name = {{ cookiecutter.package_name }}
 site_name = "{% raw %}{{ package_name }}{% endraw %}.lizard.net"
+checkout_name = please-provide-checkout-name-in-the-extra-vars
```

Comment:
Ah.... die is wel erg lief en informatief :-)

Maar dit hoort inderdaad prima te werken zo.